spec/features/users/password_recovery_via_recovery_code_spec.rb

```diff
@@ -59,7 +59,7 @@
   end
 
   scenario 'resets password, not allowed to use personal key as 2fa', email: true do
-    personal_key = personal_key_from_pii(user, pii)
+    _personal_key = personal_key_from_pii(user, pii)
```

Comment:
Do we need the assignment at all? Or is it more because the name of the method doesn't really otherwise indicate that there's a side effect?